caproto/threading/client.py

```diff
@@ -777,6 +787,7 @@ def _retry_unanswered_searches(self):
         # RETRY_RETIRED_SEARCHES_INTERVAL or, again, whenever new searches
         # are added.
         self.log.debug('Broadcaster search-retry thread has started.')
+        self.search_log.debug('Broadcaster search-retry thread has started.')
```

Comment:
Same here as above: no need to log to `caproto.bcast.search` _and_ `caproto.bcast`. I think we should leave this is as a `caproto.bcast` log message only, because logging stuff about the startup on the broadcaster machinery, not related to a search itself.